app/actors/sufia/create_with_files_actor.rb

```diff
@@ -46,10 +46,32 @@ def uploaded_files
       def mark_inactive
         return true unless Flipflop.enable_mediated_deposit?
         curation_concern.state = inactive_uri
+        send_inactive_message
       end
 
       def inactive_uri
         ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive')
       end
+
+      def send_inactive_message
+        return true if receiving_users == []
+        receiving_users.each do |admin|
+          message = "#{user.name} created <a href='#{work_path}'>#{curation_concern.title.first}</a>"
+          ::User.batch_user.send_message(admin, message, 'New work to review')
+        end
+      end
+
+      # Gather all admin users (if the admin role is defined)
+      def receiving_users
+        receivers = []
+        ::User.find_each do |user|
+          receivers << user if (user.respond_to? :admin?) && user.admin?
+        end
+        receivers
+      end
+
+      def work_path
+        Rails.application.routes.url_helpers.curation_concerns_generic_work_path(curation_concern)
```

Comment:
@mjgiarlo I'm using this path because when I tried polymorphic_path here with curation_concern it errored as an undefined method.  How can I use a polymorphic path here?